Entity Explorer - Host.ipynb

```diff
@@ -0,0 +1,766 @@
+{
```

Comment:
<p>Title - Windows and Linux host explorer or just "Host Explorer"</p><p> </p><p>One uber-point to consider - we used to have separate explorers for Linux and Windows. These are both in the Sentinel portal and prob linked to from a bunch of places.</p><p>I'm wondering if we should do something like:</p><ul><li>Make the old notebooks copies of this notebook but with a deprecation notice at the top. That way the notebooks would still work and have the benefits of the upgrade and we could transition people to the new generic notebook.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/162/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>